fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
         update.extractall(self.workspaceunpack)
 
-        # Save the rootfs filename of easy access
-        rootfsfiles = os.listdir(self.workspace + "/update/resin-root")
-        self.rootfsarchive = self.workspace + "/update/resin-root/" + os.listdir(self.workspace + "/update/resin-root")[0]
-
         log.debug("Unpacked " + self.workspacefile + " in " + self.workspaceunpack)
         return True
 
+    def unpackQuirks(self, location):
+        '''Copies the content of <quirks> directory from an update to <location>.'''
+        quirks_path = os.path.join(self.workspaceunpack, 'quirks')
+        log.info("Unpacking rootfs quirks from " + quirks_path + " to " + self.workspaceunpack + ".")
```

Comment:
According to the code, unpacking happens from `quirks_path` to `location`, not `self.workspaceunpack`